test/graphinfo.jl

```diff
@@ -36,8 +36,11 @@ A = sparse([0 0 0 0 0; 0 0 0 0 0; 0 0 0 0 0; 0 1 1 0 0; 1 0 0 1 0])
 
 # check the values from the NamedTuple match the values in the fields of GraphInfo
 vals = AbstractPPL.GraphPPL.getvals(model)
-for (i, field) in enumerate([:value, :eval, :kind])
-    @test eval( :( values(m.g.$field) == vals[$i] ) )
+for (i, v) in enumerate(vals[1])
+    @test values(m.g.value[i])[] == Ref(v)[]
```

Comment:
@PavanChaggar can we modify `values` so that it internally dereferences `Ref` values so that we can still write `values(m.g.value[i]) == v`?

Reply:
As said elsewhere, I'd completely forgo exposing the fields and have `getvalue`/`setvalue!` only.